src/account/account.service.ts

```diff
@@ -11,23 +17,26 @@ import {
     TABLE_FOLLOWS,
     TABLE_USERS,
 } from '../constants';
+import type { Account } from './account.entity';
+import type { KnexAccountRepository } from './account.repository.knex';
 import type {
-    Account,
+    Account as AccountType,
```

Comment:
This feels awkward, I think i'd have preferred aliasing the entity to to `AccountEntity`

Is this likely to be a temporary thing? Should `AccountType` and `Account` entity be the same thing?

Reply:
Yeah this is temporary, eventually `AccountType` will be completely replaced with `Account` 😬